src/icons/SealLogo.tsx

```diff
@@ -0,0 +1,59 @@
+import classnames, {
+  borderRadius,
+  boxShadow,
+  boxShadowColor,
+  fill,
+  height,
+  stroke,
+  strokeWidth,
+  width,
+} from 'classnames/tailwind'
+
+const svgClasses = classnames(
+  boxShadow('shadow-lg'),
+  boxShadowColor('shadow-secondary'),
+  borderRadius('rounded-full'),
+  width('w-16'),
+  height('h-16')
+)
+const pathClasses = classnames(
+  stroke('stroke-secondary'),
+  strokeWidth('stroke-2')
+)
+const circleClasses = classnames(pathClasses, fill('fill-primary-dark'))
+
+export default function () {
```

Comment:
Maybe let's name the file something else? Like `SealWarning`?

Reply:
Sure, why not